pages/new-project.js

```diff
@@ -0,0 +1,5 @@
+import Form from "@/components/Form";
+
+export default function NewProject() {
```

Comment:
If it's a page, I usually prefer the suffix -Page to be added. Of course keep in mind that if you do change it, you need to change the export as well.
```suggestion
export default function NewProjectPage() {
```